topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -882,7 +888,43 @@ The configuration is quite simple thanks to the many sensible defaults that are
 >    > 4. `workers` in job_conf (covered later) is the number of threads in an internal Galaxy thread pool that are available in each job handler for preparing and finishing jobs (more threads increases throughput during periods of frequent submissions or slow response times from the cluster scheduler, but there is no benefit in setting it too high due to the Python GIL).
 >    {: .tip}
 >
-> 5. Run the playbook.
+> 5. Let's set up our vault to store the secrets for these tutorials.
+>
+>    > ### {% icon code-in %} Input: Bash
+>    > ```
+>    > openssl rand -base64 24 > vault-password.txt
+>    > ```
+>    {: .code-in}
+>
+> 6. Tell Ansible where to find the decryption key. Edit your file `ansible.cfg` and add the `vault_password_file` variable.
+>
+>    ```diff
+>     [defaults]
+>     interpreter_python = /usr/bin/python3
+>     inventory = hosts
+>     retry_files_enabled = false
+>    +vault_password_file = vault-password.txt
+>    ```
+>
+> 7. Create the vault:
+>
+>    > ### {% icon code-in %} Input: Bash
+>    > ```
+>    > ansible-vault create group_vars/secret.yml
+>    > ```
+>    {: .code-in}
+>
+>    This will open the editor. Within that file, define your `id_secret` to be a long random value.
```

Comment:
Not sure if that's necessary, but I've learnt somewhere a "policy" of naming variables stored in the vault with a `vault_` prefix, to make it easier to find where they are defined. Floating the idea...

Reply:
I can support that!